GVRf/Framework/framework/src/main/java/org/gearvrf/GVRComponent.java

```diff
@@ -109,17 +121,40 @@ public boolean hasOwnerObject() {
     }
     
     /**
+     * Enable or disable this component.
+     * @param flag true to enable, false to disable.
+     * @see enable
+     * @see disable
+     * @see isEnabled
+     */
+    public void setEnable(boolean flag) {
```

Comment:
You would think so but in scanning the code I discovered enable / disable, setEnable, setEnabled, etc. I put setEnable in for backwards compatibility because some components (like GVREyePointeeHolder) have that in their API. I would like all components to at least use the same native function to enable and disable themselves instead of each component having a different way.

I agree we should settle on something. The reason I don't like enable() and disable() is that, if I am coming from an API that passes a flag, I have to write code that tests the flag and calls one of two functions. If I start out with an underlying API that takes a flag, it is easy to implement enable and disable.